src/routes/api/booking/+server.ts

```diff
@@ -115,8 +120,28 @@ export const POST = async (event) => {
 		.execute();
 
 	if (dbResults.length == 0) {
-		console.log('There is no vehicle which is able to do the tour.');
-		return json({ status: 1 });
+		try {
+			await db
+				.selectFrom('zone')
+				.where((eb) =>
+					eb.and([
+						eb('zone.is_community', '=', false),
+						sql<boolean>`ST_Covers(zone.area, ST_SetSRID(ST_MakePoint(${fromCoordinates.lng}, ${fromCoordinates.lat}),4326))`,
+						sql<boolean>`ST_Covers(zone.area, ST_SetSRID(ST_MakePoint(${toCoordinates.lng}, ${toCoordinates.lat}),4326))`
+					])
+				)
+				.executeTakeFirstOrThrow();
```

Comment:
```suggestion
				.executeTakeFirstOrThrow();
```

Kein throw als Control Flow